Bug Reporting/ARKEmailBugReporter.m

```diff
@@ -25,7 +25,7 @@
 #import "ARKDefaultLogFormatter.h"
 #import "ARKLogStore.h"
 #import "ARKLogMessage.h"
-
+#import "Aardvark.h"
```

Comment:
Why do you need this? What was missing from the above? Why not just use that instead of bringing in the umbrella header?